src/components/authorize/authorize-controller.ts

```diff
@@ -89,6 +89,13 @@ export function authorizeGet(
     req.session.client.prompt = loginPrompt;
 
     req.session.client.serviceType = claims.service_type;
+    if (claims.claim !== undefined) {
+      const claim = JSON.parse(claims.claim);
+      if (claim.userinfo !== undefined) {
+        req.session.client.claim = Object.keys(claim.userinfo);
+      }
+    }
+
     req.session.client.name = claims.client_name;
     req.session.client.scopes = claims.scope.split(" ");
```

Comment:
Basically a duplicate of the other question, but what is `claims.scope`?

Reply:
As above, good point, we can remove